test/hmr.js

```diff
@@ -61,6 +61,47 @@ describe('hmr', function() {
     assert.deepEqual(msg.assets[0].deps, {});
   });
 
+  it('should not enable HMR for --target=node', async function() {
+    await ncp(__dirname + '/integration/commonjs', __dirname + '/input');
+
+    b = bundler(__dirname + '/input/index.js', {
+      watch: true,
+      hmr: true,
+      target: 'node'
+    });
+    await b.bundle();
+
+    ws = new WebSocket('ws://localhost:' + b.options.hmrPort);
+
+    let err = await nextEvent(ws, 'error');
+    assert.equal(err.code, 'EADDRNOTAVAIL');
```

Comment:
I get `ECONNREFUSED` on my machine. Perhaps it should just assert that *some* error happen, without checking which?